src/app/application.cpp

```diff
@@ -706,7 +706,7 @@ try
 #ifndef DISABLE_GUI
     UIThemeManager::initInstance();
 
-    m_desktopIntegration = new DesktopIntegration(this);
+    m_desktopIntegration = new DesktopIntegration;
```

Comment:
May be better to use 
`m_desktopIntegration = new DesktopIntegration(m_window);`

You do not have to do explicit `delete` in `Application::cleanup()`

Reply:
Thanks

Reply:
> May be better to use
> `m_desktopIntegration = new DesktopIntegration(m_window);`

Desktop integration needs to be created earlier than main window.